python/helper-image/launcher/launcher.go

```diff
@@ -308,16 +309,23 @@ func (pc *pythonContext) updateCommandLine(ctx context.Context) error {
 		// Appropriate location to resolve pydevd is set in updateEnv
 		// TODO: check for modules (and fail?)
 		cmdline = append(cmdline, pc.args[0])
-		cmdline = append(cmdline, "-m", "pydevd", "--port", strconv.Itoa(int(pc.port)), "--server")
+		cmdline = append(cmdline, "-m", "pydevd", "--server", "--port", strconv.Itoa(int(pc.port)))
 		if pc.env["WRAPPER_VERBOSE"] != "" {
 			cmdline = append(cmdline, "--DEBUG")
 		}
-		if pc.debugMode == ModePydevdPycharm {
```

Comment:
is this just some side cleanup?

Reply:
It is — will update the description: I added this from trawling through the code, but it just seems to hang.  I figured I should remove it until you start investigating and figure out what really needs to change.